htdocs/class/captcha/recaptcha/recaptchalib.php

```diff
@@ -270,7 +270,7 @@ function recaptcha_mailhide_url($pubkey, $privkey, $email)
  */
 function _recaptcha_mailhide_email_parts($email)
 {
-    $arr = preg_split('/@/', $email);
+    $arr = explode('/@/', $email);
```

Comment:
That is actually broken, not at all equivalent.

Don't fix it, as really we should revert this file to a pristine vendor copy. It really is hard to verify that we have the latest version.